src/site/content/en/blog/commonjs-larger-bundles/index.md

```diff
@@ -0,0 +1,244 @@
+---
+title: How CommonJS is making your bundles larger
+subhead: Learn how CommonJS modules are impacting the tree-shaking of your application
+authors:
+  - mgechev
+date: 2020-05-07
+updated: 2020-05-07
+hero: hero.jpg
```

Comment:
This hero looks great on the blog index:

![image](https://user-images.githubusercontent.com/4713486/81427693-005fbd00-9110-11ea-8c95-ff971bfc72f1.png)

But unfortunately not as good on the page itself:

![image](https://user-images.githubusercontent.com/4713486/81427778-1cfbf500-9110-11ea-8795-898a4c8da14b.png)

I like the tree-shaking analogy. Can you find another image of a tree that fits the hero dimensions (3200 pixels wide, 960 pixels tall)?

If you can't find an image that looks good on both the page itself and web.dev/blog you can do art direction for the web.dev/blog image with the thumbnail field: https://web.dev/handbook/markup-media/#thumbnail

Please crop the image to 3200 by 960 and lightly optimize it with [Squoosh](https://squoosh.app). Try to reduce the file size without any noticeable reduction in image quality.

Reply:
Just updated the hero image. Let me know what you think:

<img width="1212" alt="Screen Shot 2020-05-08 at 11 10 14 AM" src="https://user-images.githubusercontent.com/455023/81435332-88e45a80-911c-11ea-8ae4-c3b46ee5d5ef.png">

<img width="1212" alt="Screen Shot 2020-05-08 at 11 10 17 AM" src="https://user-images.githubusercontent.com/455023/81435338-8c77e180-911c-11ea-9d66-644d7257d621.png">

Reply:
Perfect